includes/class-pinterest-for-woocommerce-ads-supported-countries.php

```diff
@@ -6,7 +6,8 @@
  * @since 1.0.5
  */
 
-use  Automattic\WooCommerce\Pinterest\API\Base;
+use Automattic\WooCommerce\Pinterest\API\Base;
+use \Exception;
```

Comment:
This file isn't namespaced, so this import doesn't have any effect. I also see a PHP warning generated.

```suggestion
```